Source: Upstream
Upstream: Yes
Reason: Python 3.9 compatibility fix.
diff --git a/vdirsyncer/storage/dav.py b/vdirsyncer/storage/dav.py
index 07d164f..a7e2422 100644
--- vdirsyncer/storage/dav.py
+++ vdirsyncer/storage/dav.py
@@ -124,7 +124,7 @@ def _merge_xml(items):
         return None
     rv = items[0]
     for item in items[1:]:
-        rv.extend(item.getiterator())
+        rv.extend(item.iter())
     return rv
 
 
-- 
2.28.0

